stable/openebs/values.yaml

```diff
@@ -137,33 +137,42 @@ webhook:
 
 jiva:
   image: "openebs/jiva"
-  imageTag: "1.9.0"
+  imageTag: "1.10.0"
   replicas: 3
   defaultStoragePath: "/var/openebs"
 
 cstor:
   pool:
     image: "openebs/cstor-pool"
-    imageTag: "1.9.0"
+    imageTag: "1.10.0"
   poolMgmt:
     image: "openebs/cstor-pool-mgmt"
-    imageTag: "1.9.0"
+    imageTag: "1.10.0"
   target:
     image: "openebs/cstor-istgt"
-    imageTag: "1.9.0"
+    imageTag: "1.10.0"
   volumeMgmt:
     image: "openebs/cstor-volume-mgmt"
-    imageTag: "1.9.0"
+    imageTag: "1.10.0"
 
 helper:
   image: "openebs/linux-utils"
-  imageTag: "1.9.0"
+  imageTag: "1.10.0"
+
+featureGates:
+  enabled: false
+  GPTBasedUUID:
+    enabled: false
+    featureGateFlag: "GPTBasedUUID"
+
+crd:
+  installEnabled: true
```

Comment:
n_: `crd.enableInstall` or `crd.install` or `crd.autoInstall` may be better?

Reply:
Done @kmova . Changed to `crd.enableInstall`